pytplot/store_data.py

```diff
@@ -99,9 +99,20 @@ def store_data(name, data=None, delete=False, newname=None):
                   np.nanmax(data_quants[base_data[0]].data.index)]
         df = base_data
         spec_bins = None
-    else:             
+    else:
         df = format_ydata(data['y'])            
         times = data['x']
+
+        # If given a list of datetime objects - note that the datetimes in the list should NOT be timezone aware
+        # (i.e., times should be in UTC) - convert times to seconds since epoch
+        if any(isinstance(t, datetime.datetime) for t in times):
+            for tt, time in enumerate(times):
+                times[tt] = (time-datetime.datetime(1970, 1, 1)).total_seconds()
```

Comment:
While testing this I came across an error that said you can't subtract a datetime object that has a timezone and a datetime object without a timezone.  If you give the line "datetime.datetime(1970, 1, 1)" a timezone, then it works.  

That said, it looks like there is already a function that converts python datetime objects into seconds since 1970, the "timestamp()" function:

https://docs.python.org/3/library/datetime.html

Though it says it pretty much does exactly the same thing your code has here, but it includes the timezone

(dt - datetime(1970, 1, 1, tzinfo=timezone.utc)).total_seconds()

Reply:
Interesting. Gotta love timezone issues. Fixing this now.